SU-HSDO/suhumsci
# feat(STN-174):  stylelint codeclimate update

# READY FOR REVIEW

## Summary
- adds stylelint to codeclimate.yml and removes scss-lint
- adds stylelintrc rules based on old plugins we were pulling in
- fixes all stylelint rules in other themes
- adds .stylelintignore so that codeclimate doesn't analyze entire codebase with stylelint...
- takes out scss-lint comments from our scss code
- gets codeclimate passing with new set up with develop branch ✅ https://codeclimate.com/github/SU-HSDO/suhumsci/pull/555 
- adds backstop js work here with new tasks

✅ run backstop js on old theme (looks good)
✅  add compiled CSS for other themes - there actually are no changes, so that's a great sign!

## Steps to Test
1. run `npm run test` in humsci_basic, ensure tests pass
2. run `npm i` in the root folder (you might need to delete the lock file or node_modules
3. run `npm run test` in the main root folder and make sure it passes
4. try adding a failing scss style in another theme scss folder, see that the stylelint fails
5. check out the humsci_basic readme and try out backtop js. you will need to run `npm i`

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
